### PR TITLE
 dev-libs/xml-security-c-2.0.2,  dev-libs/xerces-c-3.2.2 version bumps

### PR DESCRIPTION
**Pull request by candrews**
dev-libs/xml-security-c-2.0.2 is required for OpenSSL 1.1 compatibility
dev-libs/xml-security-c-2.0.2 requires dev-libs/xerces-c-3.2.2

### CODE REVIEW

File: dev-libs/xml-security-c/xml-security-c-2.0.2.ebuild [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+IUSE="debug examples libressl nss static-libs xalan"
+
+RDEPEND=">=dev-libs/xerces-c-3.2
+	!libressl? ( dev-libs/openssl:0= )
+	libressl? ( dev-libs/libressl:0= )
+	nss? ( dev-libs/nss )
+	xalan? ( dev-libs/xalan-c )"
+DEPEND="${RDEPEND}
+	virtual/pkgconfig"
```
Comment (Zlogene) [inline dev-libs/xml-security-c/xml-security-c-2.0.2.ebuild]: this is `BDEPEND`

Comment (candrews) [reply dev-libs/xml-security-c/xml-security-c-2.0.2.ebuild]: Thanks! I've forced pushed this update

File: dev-libs/xerces-c/xerces-c-3.2.2.ebuild [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+	curl? ( net-misc/curl )
+	virtual/libiconv"
+DEPEND="${RDEPEND}
+	doc? ( app-doc/doxygen )
+	test? ( dev-lang/perl )"
+
+DOCS=( CREDITS KEYS NOTICE README )
+
+pkg_setup() {
+	export ICUROOT="/usr"
```
Comment (SoapGentoo) [inline dev-libs/xerces-c/xerces-c-3.2.2.ebuild]: missing `${EPREFIX}`

File: dev-libs/xerces-c/xerces-c-3.2.2.ebuild [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+		doxygen || die "making docs failed"
+		HTML_DOCS=( doc/html/. )
+	fi
+}
+
+src_install () {
+	cmake-utils_src_install
+
+	# package provides .pc files
+	find "${D}" -name '*.la' -delete || die
```
Comment (SoapGentoo) [inline dev-libs/xerces-c/xerces-c-3.2.2.ebuild]: I love the insanity that upstreams go to to emulate libtool in cmake... just hilarious. Check though whether this is really still needed

File: dev-libs/xml-security-c/xml-security-c-2.0.2.ebuild [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+EAPI=7
+
+DESCRIPTION="Apache C++ XML security libraries"
+HOMEPAGE="http://santuario.apache.org/"
+SRC_URI="mirror://apache/santuario/c-library/${P}.tar.gz"
+
+LICENSE="Apache-2.0"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
```
Comment (SoapGentoo) [inline dev-libs/xml-security-c/xml-security-c-2.0.2.ebuild]: remove this empty line

File: dev-libs/xml-security-c/xml-security-c-2.0.2.ebuild [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+IUSE="debug examples libressl nss static-libs xalan"
+
+RDEPEND=">=dev-libs/xerces-c-3.2
+	!libressl? ( dev-libs/openssl:0= )
+	libressl? ( dev-libs/libressl:0= )
+	nss? ( dev-libs/nss )
+	xalan? ( dev-libs/xalan-c )"
+DEPEND="${RDEPEND}"
+BDEPEND="virtual/pkgconfig"
```
Comment (SoapGentoo) [inline dev-libs/xml-security-c/xml-security-c-2.0.2.ebuild]: :+1:

File: dev-libs/xml-security-c/xml-security-c-2.0.2.ebuild (hunk 4 above)
Comment (candrews) [reply dev-libs/xml-security-c/xml-security-c-2.0.2.ebuild]: :+1:

File: dev-libs/xerces-c/xerces-c-3.2.2.ebuild (hunk 3 above)
Comment (candrews) [reply dev-libs/xerces-c/xerces-c-3.2.2.ebuild]: seems to be... the `.la` files still exist.

File: dev-libs/xerces-c/xerces-c-3.2.2.ebuild (hunk 2 above)
Comment (candrews) [reply dev-libs/xerces-c/xerces-c-3.2.2.ebuild]: :+1:

**Pull request closed without merging**